[CI] Run perf test for perf_cpu_aarch64

desertfire opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack) (oldest at bottom):
* __->__ #132038

Summary: Run perf test for perf_cpu_aarch64 instead of regular CI test (test_linux_aarch64).

desertfire committed:
[CI] Run perf test for perf_cpu_aarch64

Summary: Run perf test for perf_cpu_aarch64 instead of regular CI test (test_linux_aarch64).

[ghstack-poisoned]

malfet reviewed (approved):
LGTM. Not an expert in shell scripting, but pattern I've seen in the past is usually  `[[ cond1 && cond2 ]];` rather than `[[ cond1 ]] && [[ cond2 ]];`

desertfire committed:
Update on "[CI] Run perf test for perf_cpu_aarch64"


Summary: Run perf test for perf_cpu_aarch64 instead of regular CI test (test_linux_aarch64).

[ghstack-poisoned]

desertfire commented:
@pytorchbot merge -f "only affects *_perf_cpu_aarch64 runs"
